apps/src/music/views/MusicView.jsx

```diff
@@ -527,6 +529,8 @@ class UnconnectedMusicView extends React.Component {
     this.player.stopSong();
     this.playingTriggers = [];
 
+    this.props.updateValidationPass();
```

Comment:
Instead of forcing an update through a key change, maybe we could instead just expose the `resetValidation` method on `ProgressManager` and have music lab call that directly? I imagine it would generally be useful for labs to reset validation conditions whenever they need to

Reply:
Thanks for the offline discussion about this.  

The new implementation calls `resetValidation`, though it's done from `UpdateTimer` since it's a functional React component that has access to `ProgressManagerContext`.  Ideally, we'd call `resetValidation` from `MusicView`, but we'll need to do this after it's converted to a functional React component.

We do keep track of an index that's incremented each time we reset validations.  That's so that we can use it as part of the `key` for the rendered feedback, to ensure that the appearance animation is played even if the same feedback was showing from the last run.